### ISSUE TITLE
Dead Nodes that come back online after ping

### ISSUE THREAD

**Issue by kars85** [AlCalzone/node-zwave-js#5088]
### Is your problem within Home Assistant (Core or Z-Wave JS Integration)?

YES, BUT a Home Assistant developer has told me to come here

### Is your problem within ZWaveJS2MQTT?

NO, my problem is NOT within ZWaveJS2MQTT

### Checklist

- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there.

### Describe the bug

What causes the bug?
Unknown, random times nodes show as dead. There is no way to reproduce the issue consistently
What do you observe?

What did you expect to happen?
Stable behavior.

Node becomes unresponsive, ping with in zjs2mqtt, and node comes back online.

Dead nodes 6 & 7 began on 9/18 and 9/19 I pinged the devices back to healthy.

### Device information

Manufacturer: GE
Model name: 28167 / ZW3104 
Node ID in your network: 6 & 7

### How are you using `node-zwave-js`?

- [X] `zwavejs2mqtt` Docker image (latest)
- [ ] `zwavejs2mqtt` Docker image (dev)
- [ ] `zwavejs2mqtt` Docker manually built _(please specify branches)_
- [ ] `ioBroker.zwave2` adapter _(please specify version)_
- [ ] `HomeAssistant zwave_js` integration  _(please specify version)_
- [ ] `pkg`
- [ ] `node-red-contrib-zwave-js` _(please specify version, double click node to find out)_
- [ ] Manually built from GitHub _(please specify branch)_
- [ ] Other _(please describe)_

### Which branches or versions?

zwavejs2mqtt: 7.1.0
zwave-js: 10.1.0

### Did you change anything?

yes (please describe)

### If yes, what did you change?

I updated my Zooz stick to 7.18.1, but this issue was present on 7.17.2 as well.

### Did this work before?

No, it never worked anywhere

### Attach Driver Logfile

[zwavejs_2022-09-18.log](https://github.com/zwave-js/node-zwave-js/files/9600489/zwavejs_2022-09-18.log)
[zwavejs_2022-09-19.log](https://github.com/zwave-js/node-zwave-js/files/9600490/zwavejs_2022-09-19.log)

**Comment by collse** [comment]
I have the same issue with one device (same firmware, version as the other that does not show the behaviour on the same network) with my aeotec z stick 7 700 - at the moment, I got an automation running to ping the device on a schedule to overcome this as I have not found a solution. I did reset the entire network already, reset all devices to defaults before re-adding etc - did not seem to help at all.  I did not upgrade to 7.18.1 yet however as I couldnt find the update yet ..

mine is a Shenzhen Neo Electronics Co., Ltd. / Light Switch 2 Channel (https://devices.zwave-js.io/?jumpTo=0x0258:0x0003:0x008b:0.0) but as mentioned the other 2 on my network, same revision, firmware etc dont show the behaviour.

**Comment by kars85** [comment]
Node 6 went dead again on 9/20 @ 9:15PM and was able to ping it back to life 9/22 @ 7:49AM. 
[zwavejs_2022-09-20.log](https://github.com/zwave-js/node-zwave-js/files/9625848/zwavejs_2022-09-20.log)
[node_6_sniff.zlf.txt](https://github.com/zwave-js/node-zwave-js/files/9625852/node_6_sniff.zlf.txt)

**Comment by jcam** [comment]
I see this as well with some devices. Can dig up logs if necessary.

**Comment by kars85** [comment]
> I see this as well with some devices. Can dig up logs if necessary.

I'll defer to @AlCalzone on if that's his preference, or that you create your own bug report (I presume that's probably his preference). Did you update your stick's FW yet?

**Comment by jcam** [comment]
I did. When I was on 1.16.x I would lose something like 90% of my nodes every few days once the firmware version/update code was in place. I upgraded to 1.17.2, and now I lose maybe one node a week that needs a ping to bring it back. My guess is that my host hardware is too slow and messages end up getting lost in the name of network congestion, as everything works much better when i move the stick to my laptop, but I'll be curious what you all come up with for this ticket.

**Comment by AlCalzone** [comment]
@kars85 your issue is most likely a reception issue of some nodes. It starts at 9:14:39 (`2022-09-21T02:14:39.121Z`) with you locking Node 15. That command takes 10 seconds to be received, but meanwhile several nodes send commands they expect a response to. However the controller cannot respond while it is busy sending to Node 15. The response is actually received well before that, but the ACK isn't, so the send cycle cannot continue.

This essentially causes a couple of nodes to repeat their messages, leading to a flood of incoming commands, eventually causing the controller to be unable to send due too much RF noise. Node 6 going "dead" is just a side effect. It isn't actually dead, just cannot be reached because of the controller being jammed. I should fix this https://github.com/zwave-js/node-zwave-js/issues/4191 so this situation is handled better, but the main reason this is happening to you is a suboptimal mesh.

You'll want to measure and improve the link quality throughout your mesh to avoid this issue altogether:
https://zwave-js.github.io/node-zwave-js/#/troubleshooting/network-health?id=testing-the-connection-strength

**Comment by AlCalzone** [comment]
@jcam try to do the same first, then I can look at some logs

**Comment by jcam** [comment]
Yes I think you're right that it is just RF noise / bad link coupled with my host being slow so that the controller takes longer to do things than it otherwise would... like, the initial startup interview takes 3-5 minutes on my mini pc, and it completes the same thing in under a minute when the stick and node are running from my laptop.
I'm going to shift the node/home assistant docker containers to be running from my NAS instead, with the stick using remote usb, but that has to wait for some other projects around the house to get finished. For now it is reliable enough :)

**Comment by AlCalzone** [comment]
> initial startup interview takes 3-5 minutes

Uhh, just pinging a handful of nodes?

**Comment by jcam** [comment]
yeah, the thing that starts with "[Node 001] Embedded device config loaded" and ends with "Network scan complete. Found: 58 nodes" is over 3 minutes. 3 of the nodes specifically take 20-30 seconds each, despite each being 10 feet from the stick, just doing dozens and dozens of "value added" things... they have 4 multilevel switch entities integrated in a single device each.
I notice that I had 59 devices until a few days ago... will have to investigate how it forgot about one of my wall switches...

**Comment by AlCalzone** [comment]
Can you show me that?

**Comment by jcam** [comment]
just did a log capture, let me know if you need more detail
[zwave-js-ui-store.zip](https://github.com/zwave-js/node-zwave-js/files/9628637/zwave-js-ui-store.zip)

**Comment by AlCalzone** [comment]
@jcam driver logs that are not on "debug" are useless for troubleshooting.

**Comment by jcam** [comment]
[zwave-js-ui-store (1).zip](https://github.com/zwave-js/node-zwave-js/files/9634800/zwave-js-ui-store.1.zip)

with debug turned on (may have the morning's non-debug first)

**Comment by AlCalzone** [comment]
And now there are no driver logs, only application logs (with UI in the name 🙈)

**Comment by jcam** [comment]
Someday I'll figure out this interface lol
[zwave-js-ui-store (2).zip](https://github.com/zwave-js/node-zwave-js/files/9634876/zwave-js-ui-store.2.zip)

try this one :)

**Comment by AlCalzone** [comment]
Ok the application logs are definitely interesting here. Looks like zui does a ton of work after Node 2 is ready (the driver logs are logged afterwards):
** Driver **
```
2022-09-23T14:47:31.754Z CNTRLR   [Node 002] The node is alive.

... zui goes brr...

2022-09-23T14:47:51.187Z CNTRLR   [Node 002] The node is ready to be used
``` 

** UI **
```
2022-09-23 10:47:31.752 INFO Z-WAVE: Node 2 is alive
2022-09-23 10:47:33.182 INFO Z-WAVE: Node 2: value added 2-91-0-scene-001 => undefined
2022-09-23 10:47:33.185 INFO Z-WAVE: Node 2: value added 2-91-0-scene-002 => undefined
2022-09-23 10:47:33.189 INFO Z-WAVE: Node 2: value added 2-91-0-scene-003 => undefined
2022-09-23 10:47:33.192 INFO Z-WAVE: Node 2: value added 2-91-0-scene-004 => undefined
2022-09-23 10:47:33.195 INFO Z-WAVE: Node 2: value added 2-91-0-slowRefresh => undefined
2022-09-23 10:47:33.198 INFO Z-WAVE: Node 2: value added 2-112-0-1 => 0
2022-09-23 10:47:33.202 INFO Z-WAVE: Node 2: value added 2-112-0-20 => 2
2022-09-23 10:47:33.205 INFO Z-WAVE: Node 2: value added 2-112-0-21 => 2
2022-09-23 10:47:33.208 INFO Z-WAVE: Node 2: value added 2-112-0-22 => 2
2022-09-23 10:47:33.211 INFO Z-WAVE: Node 2: value added 2-112-0-23 => 2
2022-09-23 10:47:33.214 INFO Z-WAVE: Node 2: value added 2-112-0-62 => 3600
2022-09-23 10:47:33.217 INFO Z-WAVE: Node 2: value added 2-112-0-65 => 10
2022-09-23 10:47:33.220 INFO Z-WAVE: Node 2: value added 2-112-0-66 => 3600
2022-09-23 10:47:33.223 INFO Z-WAVE: Node 2: value added 2-112-0-150 => 0
2022-09-23 10:47:33.227 INFO Z-WAVE: Node 2: value added 2-112-0-151 => 3
2022-09-23 10:47:33.230 INFO Z-WAVE: Node 2: value added 2-112-0-152 => 3
2022-09-23 10:47:33.233 INFO Z-WAVE: Node 2: value added 2-112-0-157 => 0
2022-09-23 10:47:33.236 INFO Z-WAVE: Node 2: value added 2-112-0-40-1 => 1
2022-09-23 10:47:33.239 INFO Z-WAVE: Node 2: value added 2-112-0-40-2 => 1
2022-09-23 10:47:33.242 INFO Z-WAVE: Node 2: value added 2-112-0-40-4 => 1
2022-09-23 10:47:33.245 INFO Z-WAVE: Node 2: value added 2-112-0-40-8 => 1
2022-09-23 10:47:33.248 INFO Z-WAVE: Node 2: value added 2-112-0-41-1 => 1
2022-09-23 10:47:33.251 INFO Z-WAVE: Node 2: value added 2-112-0-41-2 => 1
2022-09-23 10:47:33.253 INFO Z-WAVE: Node 2: value added 2-112-0-41-4 => 1
2022-09-23 10:47:33.256 INFO Z-WAVE: Node 2: value added 2-112-0-41-8 => 1
2022-09-23 10:47:33.259 INFO Z-WAVE: Node 2: value added 2-112-0-42-1 => 1
2022-09-23 10:47:33.262 INFO Z-WAVE: Node 2: value added 2-112-0-42-2 => 1
2022-09-23 10:47:33.265 INFO Z-WAVE: Node 2: value added 2-112-0-42-4 => 1
2022-09-23 10:47:33.268 INFO Z-WAVE: Node 2: value added 2-112-0-42-8 => 1
2022-09-23 10:47:33.271 INFO Z-WAVE: Node 2: value added 2-112-0-43-1 => 1
2022-09-23 10:47:33.274 INFO Z-WAVE: Node 2: value added 2-112-0-43-2 => 1
2022-09-23 10:47:33.277 INFO Z-WAVE: Node 2: value added 2-112-0-43-4 => 1
2022-09-23 10:47:33.280 INFO Z-WAVE: Node 2: value added 2-112-0-43-8 => 1
2022-09-23 10:47:33.283 INFO Z-WAVE: Node 2: value added 2-114-0-manufacturerId => 634
2022-09-23 10:47:33.285 INFO Z-WAVE: Node 2: value added 2-114-0-productType => 2306
2022-09-23 10:47:33.288 INFO Z-WAVE: Node 2: value added 2-114-0-productId => 8192
2022-09-23 10:47:33.291 INFO Z-WAVE: Node 2: value added 2-134-0-libraryType => 3
2022-09-23 10:47:33.295 INFO Z-WAVE: Node 2: value added 2-134-0-protocolVersion => 6.4
2022-09-23 10:47:33.297 INFO Z-WAVE: Node 2: value added 2-134-0-firmwareVersions => 5.0,5.0,1.0
2022-09-23 10:47:33.300 INFO Z-WAVE: Node 2: value added 2-134-0-hardwareVersion => 2
2022-09-23 10:47:33.304 INFO Z-WAVE: Node 2: value added 2-38-1-targetValue => 0
2022-09-23 10:47:33.307 INFO Z-WAVE: Node 2: value added 2-38-1-duration => 0s
2022-09-23 10:47:34.709 INFO Z-WAVE: Node 2: value added 2-38-1-currentValue => 0
2022-09-23 10:47:34.713 INFO Z-WAVE: Node 2: value added 2-38-1-Up => undefined
2022-09-23 10:47:34.717 INFO Z-WAVE: Node 2: value added 2-38-1-Down => undefined
2022-09-23 10:47:34.720 INFO Z-WAVE: Node 2: value added 2-38-1-restorePrevious => undefined
2022-09-23 10:47:34.724 INFO Z-WAVE: Node 2: value added 2-50-1-value-65537 => 4.05
2022-09-23 10:47:34.727 INFO Z-WAVE: Node 2: value added 2-50-1-value-66049 => 0
2022-09-23 10:47:34.730 INFO Z-WAVE: Node 2: value added 2-50-1-reset => undefined
2022-09-23 10:47:34.733 INFO Z-WAVE: Node 2: value added 2-51-1-duration => 0s
2022-09-23 10:47:36.029 INFO Z-WAVE: Node 2: value added 2-51-1-currentColor-0 => 255
2022-09-23 10:47:36.033 INFO Z-WAVE: Node 2: value added 2-51-1-targetColor-0 => 0
2022-09-23 10:47:37.271 INFO Z-WAVE: Node 2: value added 2-51-1-currentColor => [object Object]
2022-09-23 10:47:37.275 INFO Z-WAVE: Node 2: value added 2-51-1-targetColor => [object Object]
2022-09-23 10:47:38.517 INFO Z-WAVE: Node 2: value added 2-51-1-currentColor-2 => 159
2022-09-23 10:47:38.521 INFO Z-WAVE: Node 2: value added 2-51-1-targetColor-2 => 0
2022-09-23 10:47:38.524 INFO Z-WAVE: Node 2: value added 2-51-1-hexColor => 9fe7ff
2022-09-23 10:47:39.761 INFO Z-WAVE: Node 2: value added 2-51-1-currentColor-3 => 231
2022-09-23 10:47:39.764 INFO Z-WAVE: Node 2: value added 2-51-1-targetColor-3 => 0
2022-09-23 10:47:41.004 INFO Z-WAVE: Node 2: value added 2-51-1-currentColor-4 => 255
2022-09-23 10:47:41.008 INFO Z-WAVE: Node 2: value added 2-51-1-targetColor-4 => 0
2022-09-23 10:47:41.012 INFO Z-WAVE: Node 2: value added 2-113-1-Power Management-Over-current status => 0
2022-09-23 10:47:41.016 INFO Z-WAVE: Node 2: value added 2-113-1-System-Hardware status => 0
2022-09-23 10:47:41.019 INFO Z-WAVE: Node 2: value added 2-113-1-alarmType => undefined
2022-09-23 10:47:41.023 INFO Z-WAVE: Node 2: value added 2-113-1-alarmLevel => undefined
2022-09-23 10:47:41.026 INFO Z-WAVE: Node 2: value added 2-117-1-local => 0
2022-09-23 10:47:41.030 INFO Z-WAVE: Node 2: value added 2-117-1-rf => 0
2022-09-23 10:47:41.033 INFO Z-WAVE: Node 2: value added 2-117-1-exclusiveControlNodeId => undefined
2022-09-23 10:47:41.037 INFO Z-WAVE: Node 2: value added 2-117-1-timeout => undefined
2022-09-23 10:47:41.040 INFO Z-WAVE: Node 2: value added 2-38-2-targetValue => 0
2022-09-23 10:47:41.044 INFO Z-WAVE: Node 2: value added 2-38-2-duration => 0s
2022-09-23 10:47:42.286 INFO Z-WAVE: Node 2: value added 2-38-2-currentValue => 0
2022-09-23 10:47:42.290 INFO Z-WAVE: Node 2: value added 2-38-2-Up => undefined
2022-09-23 10:47:42.294 INFO Z-WAVE: Node 2: value added 2-38-2-Down => undefined
2022-09-23 10:47:42.297 INFO Z-WAVE: Node 2: value added 2-38-2-restorePrevious => undefined
2022-09-23 10:47:42.301 INFO Z-WAVE: Node 2: value added 2-38-3-targetValue => 0
2022-09-23 10:47:42.304 INFO Z-WAVE: Node 2: value added 2-38-3-duration => 0s
2022-09-23 10:47:43.570 INFO Z-WAVE: Node 2: value added 2-38-3-currentValue => 0
2022-09-23 10:47:43.574 INFO Z-WAVE: Node 2: value added 2-38-3-Up => undefined
2022-09-23 10:47:43.578 INFO Z-WAVE: Node 2: value added 2-38-3-Down => undefined
2022-09-23 10:47:43.581 INFO Z-WAVE: Node 2: value added 2-38-3-restorePrevious => undefined
2022-09-23 10:47:43.585 INFO Z-WAVE: Node 2: value added 2-38-4-targetValue => 0
2022-09-23 10:47:43.588 INFO Z-WAVE: Node 2: value added 2-38-4-duration => 0s
2022-09-23 10:47:44.846 INFO Z-WAVE: Node 2: value added 2-38-4-currentValue => 0
2022-09-23 10:47:44.850 INFO Z-WAVE: Node 2: value added 2-38-4-Up => undefined
2022-09-23 10:47:44.853 INFO Z-WAVE: Node 2: value added 2-38-4-Down => undefined
2022-09-23 10:47:44.856 INFO Z-WAVE: Node 2: value added 2-38-4-restorePrevious => undefined
2022-09-23 10:47:44.860 INFO Z-WAVE: Node 2: value added 2-38-5-targetValue => 0
2022-09-23 10:47:44.863 INFO Z-WAVE: Node 2: value added 2-38-5-duration => 0s
2022-09-23 10:47:46.095 INFO Z-WAVE: Node 2: value added 2-38-5-currentValue => 0
2022-09-23 10:47:46.099 INFO Z-WAVE: Node 2: value added 2-38-5-Up => undefined
2022-09-23 10:47:46.102 INFO Z-WAVE: Node 2: value added 2-38-5-Down => undefined
2022-09-23 10:47:46.105 INFO Z-WAVE: Node 2: value added 2-38-5-restorePrevious => undefined
2022-09-23 10:47:47.345 INFO Z-WAVE: Node 2: value added 2-32-6-currentValue => 0
2022-09-23 10:47:47.349 INFO Z-WAVE: Node 2: value added 2-32-6-targetValue => undefined
2022-09-23 10:47:47.353 INFO Z-WAVE: Node 2: value added 2-32-6-duration => NaNs
2022-09-23 10:47:47.356 INFO Z-WAVE: Node 2: value added 2-32-6-restorePrevious => undefined
2022-09-23 10:47:47.360 INFO Z-WAVE: Node 2: value added 2-49-6-Voltage => 0
2022-09-23 10:47:48.605 INFO Z-WAVE: Node 2: value added 2-32-7-currentValue => 0
2022-09-23 10:47:48.609 INFO Z-WAVE: Node 2: value added 2-32-7-targetValue => undefined
2022-09-23 10:47:48.613 INFO Z-WAVE: Node 2: value added 2-32-7-duration => NaNs
2022-09-23 10:47:48.616 INFO Z-WAVE: Node 2: value added 2-32-7-restorePrevious => undefined
2022-09-23 10:47:48.620 INFO Z-WAVE: Node 2: value added 2-49-7-Voltage => 0
2022-09-23 10:47:49.856 INFO Z-WAVE: Node 2: value added 2-32-8-currentValue => 0
2022-09-23 10:47:49.860 INFO Z-WAVE: Node 2: value added 2-32-8-targetValue => undefined
2022-09-23 10:47:49.864 INFO Z-WAVE: Node 2: value added 2-32-8-duration => NaNs
2022-09-23 10:47:49.867 INFO Z-WAVE: Node 2: value added 2-32-8-restorePrevious => undefined
2022-09-23 10:47:49.871 INFO Z-WAVE: Node 2: value added 2-49-8-Voltage => 0
2022-09-23 10:47:51.103 INFO Z-WAVE: Node 2: value added 2-32-9-currentValue => 0
2022-09-23 10:47:51.107 INFO Z-WAVE: Node 2: value added 2-32-9-targetValue => undefined
2022-09-23 10:47:51.110 INFO Z-WAVE: Node 2: value added 2-32-9-duration => NaNs
2022-09-23 10:47:51.114 INFO Z-WAVE: Node 2: value added 2-32-9-restorePrevious => undefined
2022-09-23 10:47:51.118 INFO Z-WAVE: Node 2: value added 2-49-9-Voltage => 0
2022-09-23 10:47:51.185 INFO Z-WAVE: Node 2 ready: Zooz - ZEN31 (RGBW Dimmer)
```

**Comment by AlCalzone** [comment]
@robertsLando this one may be interesting for you. We're definitely off-topic for the original issue, I'll make a new one. --> https://github.com/zwave-js/zwave-js-ui/issues/2676

**Comment by collse** [comment]
can someone point me to the latest version for the aeotec z stick 7 700 they seemed to change their naming convention for the update files so its not as obvious as it could be

**Comment by kars85** [comment]
The same nodes (6 & 7) dropped offline again yesterday. This was after a heal a couple days prior. Attached are the silly logs and zniffer during that timeframe.
[zwavejs_2022-09-27.log](https://github.com/zwave-js/node-zwave-js/files/9669736/zwavejs_2022-09-27.log)
[node_6_and7_sniff.txt](https://github.com/zwave-js/node-zwave-js/files/9669744/node_6_and7_sniff.txt)

Change .txt to .zip and open to expose .zlf

**Comment by AlCalzone** [comment]
@kars85 similar issue, different node this time. At `2022-09-28T04:00:00.633Z`, a **Hail** is received from nodes 19, 23 and 24 (immediately after you turned them off). This was an old way to get around a patent preventing Z-Wave from unsolicited status reports. In response to the Hail, the driver queries these 3 nodes for updated values.

This time, Node 24's command gets repeated over and over, essentially RF jamming the controller:
```
2022-09-28T04:00:01.538Z DRIVER « [Node 024] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -77 dBm
                                  └─[HailCC]
2022-09-28T04:00:01.539Z CNTRLR   [Node 024] Hail received from node, but still busy with previous one...
2022-09-28T04:00:01.596Z SERIAL « 0x010b00a800011802820100b377                                        (13 bytes)
2022-09-28T04:00:01.597Z SERIAL » [ACK]                                                                   (0x06)
2022-09-28T04:00:01.597Z DRIVER « [Node 024] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -77 dBm
                                  └─[HailCC]
2022-09-28T04:00:01.598Z CNTRLR   [Node 024] Hail received from node, but still busy with previous one...
2022-09-28T04:00:01.621Z SERIAL « 0x010b00a800011802820100b377                                        (13 bytes)
2022-09-28T04:00:01.621Z SERIAL » [ACK]                                                                   (0x06)
2022-09-28T04:00:01.621Z DRIVER « [Node 024] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -77 dBm
                                  └─[HailCC]
2022-09-28T04:00:01.622Z CNTRLR   [Node 024] Hail received from node, but still busy with previous one...
2022-09-28T04:00:01.713Z SERIAL « 0x010b00a800011802820100af6b                                        (13 bytes)
2022-09-28T04:00:01.713Z SERIAL » [ACK]                                                                   (0x06)
2022-09-28T04:00:01.713Z DRIVER « [Node 024] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -81 dBm
                                  └─[HailCC]
2022-09-28T04:00:01.714Z CNTRLR   [Node 024] Hail received from node, but still busy with previous one...

... 4 seconds later ...

2022-09-28T04:00:05.325Z DRIVER « [Node 024] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -84 dBm
                                  └─[HailCC]
2022-09-28T04:00:05.325Z CNTRLR   [Node 024] Hail received from node, but still busy with previous one...
2022-09-28T04:00:05.352Z SERIAL « 0x010b00a800011802820100ab6f                                        (13 bytes)
2022-09-28T04:00:05.352Z SERIAL » [ACK]                                                                   (0x06)
2022-09-28T04:00:05.352Z DRIVER « [Node 024] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -85 dBm
                                  └─[HailCC]
2022-09-28T04:00:05.353Z CNTRLR   [Node 024] Hail received from node, but still busy with previous one...
2022-09-28T04:00:05.415Z SERIAL « 0x010b00a800011802820100ad69                                        (13 bytes)
2022-09-28T04:00:05.415Z SERIAL » [ACK]                                                                   (0x06)
2022-09-28T04:00:05.415Z DRIVER « [Node 024] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -83 dBm
                                  └─[HailCC]
2022-09-28T04:00:05.416Z CNTRLR   [Node 024] Hail received from node, but still busy with previous one...
2022-09-28T04:00:05.486Z SERIAL « 0x010b00a800011802820100af6b                                        (13 bytes)
2022-09-28T04:00:05.486Z SERIAL » [ACK]                                                                   (0x06)
2022-09-28T04:00:05.487Z DRIVER « [Node 024] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -81 dBm
                                  └─[HailCC]
2022-09-28T04:00:05.487Z CNTRLR   [Node 024] Hail received from node, but still busy with previous one...

... etc

```

There are also some pretty shitty routes in the mesh, e.g. for this command to node 32:
```
transmit status:        OK, took 970 ms
protocol & route speed: Z-Wave, 9.6 kbit/s
```
9.6 kbps is the last resort when nothing else works.

Did you follow my suggestion to do this:

> You'll want to measure and improve the link quality throughout your mesh to avoid this issue altogether:
> [zwave-js.github.io/node-zwave-js/#/troubleshooting/network-health?id=testing-the-connection-strength](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/network-health?id=testing-the-connection-strength)

?

**Comment by AlCalzone** [comment]
Here's the same visible in your Zniffer trace:
![grafik](https://user-images.githubusercontent.com/17641229/192967527-798e7080-d272-49bc-b3f9-a90227fc3e82.png)

Node 24 sends the Hail via different routes:
- 24 > 6 > 1 @ 40k
- 24 > 4 > 1 @ 100k (2x)
- 24 > 4 > 1 @ 40k (3x)
In the Zniffer trace it looks like 24 doesn't get an ACK from the repeater nodes for any of those, but that might just be the Zniffer placement. Although it would explain why Node 24 repeats its command so often.

Then it tries via node 8 as a repeater, for which the Zniffer picks up that the commands reached Node 1. For node 8 we can also see that it thinks it failed to reach node 1 and tries to tell that to node 24 (Routed Error).

Later (not sure what you did there), I see a lot of Routed Errors again where node 24 is involved:
![grafik](https://user-images.githubusercontent.com/17641229/192969328-17a70a7e-64b2-479d-8df8-1444defd1811.png)

**Comment by AlCalzone** [comment]
TL;DR: Do the health checks for your entire network. Work your way outwards from the controller by distance, and if there's any check that's not ideal, fix it.
